test/test_components.py

```diff
@@ -1895,6 +1895,16 @@ def test_component_functions(self):
             "_selectable": False,
         }
 
+    def test_chatbot_selectable_in_config(self):
```

Comment:
```suggestion
    def test_chatbot_likeable_in_config(self):
```

Reply:
Maybe we can also test to ensure that likable is _not_ present if no `.like` event has been set

Reply:
Yea good idea!